app/__init__.py

```diff
@@ -4,6 +4,10 @@
 from flask_compress import Compress
 from flask_sqlalchemy import SQLAlchemy
 from flask_humanize import Humanize
+import os.path
+import sys
+
+sys.path.append( os.path.dirname( os.path.abspath( __file__ ) ) )
```

Comment:
I never needed that so far. Is this to make the code robust against the working directory?

Reply:
Yes, this line is for ```create_db.py``` to be working when launched exactly from the root directory of the repository. I had problems with ```from config import ...``` Python couldn't find module ```config```. And when I added this line to ```app/__init__.py ```create_db.py``` became working.

Reply:
Sure, user can tune python's environment so all will work without this line. But I'm not sure about ```from config import ....```, why not ```from app.config import ....```?